rkt/list.go

```diff
@@ -102,10 +134,115 @@ func runList(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
+	if len(errors) > 0 {
+		sep := "----------------------------------------"
+		stderr("%d error(s) encountered when listing pods:", len(errors))
+		stderr("%s", sep)
+		for _, err := range errors {
+			stderr("%s", err.Error())
+			stderr("%s", sep)
+		}
+		stderr("Misc:")
+		stderr("  rkt's appc version: %s", schema.AppContainerVersion)
+		stderr("%s", sep)
+		// make a visible break between errors and the listing
+		stderr("")
+	}
+
 	tabOut.Flush()
+	stdout("%s", tabBuffer.String())
 	return 0
 }
 
+func newPodListReadError(p *pod, err error) error {
+	lines := []string{
+		fmt.Sprintf("Unable to read pod %s manifest:", p.uuid.String()),
+		fmt.Sprintf("  %v", err),
+	}
+	return fmt.Errorf("%s", strings.Join(lines, "\n"))
+}
+
+func newPodListLoadError(p *pod, err error, pmj []byte) error {
+	lines := []string{
+		fmt.Sprintf("Unable to load pod %s manifest, because it is invalid:", p.uuid.String()),
```

Comment:
Should `p.uuid` be printed by following `flagFullOutput` as it is done in 
[list.go#L79](https://github.com/coreos/rkt/blob/master/rkt/list.go#L79)?